### DISCUSSION TITLE
ability for a regex match for an input field to be valid

### DISCUSSION THREAD

**Discussion by ThatGravyBoat** [community/community#4629] [Projects and Issues]
This feature would be amazing as you can force the user to put in the correct format for a version id than them putting some random text.
A scenario where this would be useful :
Repo owner wants to know the version id of the software they're using which is in a format like this `1.0.25` but the user only puts in `25` it would stop cases where the person looking at the issue doesn't know all the context as only part of the info was given to them.

**Comment by lukehefson** [comment]
Thanks for the suggestion @ThatGravyBoat!

Yes, this is something that we’ve considered a little in the past and I think it could be really cool. Definitely something that we might consider for a future iteration! Your use-case is really helpful. Thanks again!
